bin/slt.js

```diff
@@ -14,7 +18,7 @@ if (cmd in tools) {
   else
     defaultCLI(tools[cmd], pkg);
 } else {
-  usage(path.basename(process.argv[1]), console.log.bind(console));
+  usage(process.env.ARGV0 || path.basename(process.argv[1]), console.log);
```

Comment:
For overriding the name of the binary given in the usage statement. Allows a wrapper to provide it's own name instead of `my-wrapper --help` telling people to run `slt ...` in the instructions.